pages/docs/parameters/parameters.md

```diff
@@ -239,11 +239,22 @@ These are parameters that tune the solver used to solve the MIP problem.
 These parameters (topological-actions-optimization) tune the [search-tree algorithm](/docs/engine/ra-optimisation/search-tree-rao) 
 when searching for the best network actions.
 
-### max-search-tree-depth {#max-search-tree-depth}
```

Comment:
the anchor "max-search-tree-depth" is used in some links in this page and in other pages. update them